server/registry/resources.py

```diff
@@ -41,6 +41,11 @@ def get_dogs(self):
 	def add_dog(self):
 		self.all_dogs.append(self)
 
+    def get_dog(id1, self):
+		for dog in self.all_dogs:
+			if dog.id == id1:
+				return {"name":dog.name,"id":dog.id}

```

Comment:
You should return the actual dog object here. And then create a web api for getting the actual dog as this json